src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -67,7 +66,7 @@ baseDaoContract config@Config{..} = defaultContract $ contractName cDaoName $ do
       )
 
 fa2Handler
-  :: (IsoValue ce, IsoValue pm, HasFuncContext s (BaseDAO.Storage ce pm))
+  :: (IsoValue ce, KnownValue pm, HasFuncContext s (BaseDAO.Storage ce pm))
```

Comment:
Why do we have to change `IsoValue` to `KnownValue`?

Reply:
There were some reorganiztion in `StoreClass`, and `KnownValue` constraint was moved to `instance StoreHasSubmap BigMap`, so serving `sProposals` map now requires `KnownValue` from `pm` :pled: